client/lib/signup/step-actions/index.js

```diff
@@ -428,63 +428,51 @@ export async function setStoreFeatures( callback, { siteSlug }, stepProvidedItem
-					<h2 class="has-text-align-left has-x-large-font-size" id="item-name">${ translate(
-						'Item Name'
-					) }</h2>
-					<!-- /wp:heading -->
-
-					<!-- wp:paragraph {"align":"left","style":{"color":{"text":"#808080"}},"fontSize":"small"} -->
-					<p class="has-text-align-left has-text-color has-small-font-size" style="color:#808080">${ translate(
-						'Quick details'
-					) }</p>
-					<!-- /wp:paragraph -->
-
-					<!-- wp:heading {"level":3,"fontSize":"medium"} -->
-					<h3 class="has-medium-font-size" id="0-00">$0.00</h3>
-					<!-- /wp:heading -->
-
-					<!-- wp:jetpack/recurring-payments -->
-					<div class="wp-block-jetpack-recurring-payments">
-						<!-- wp:jetpack/button {"element":"a","uniqueId":"recurring-payments-id","text":"${ translate(
-							'Buy Now'
-						) }","backgroundColor":"primary","borderRadius":0,"width":"100%"} /-->
-					</div>
-					<!-- /wp:jetpack/recurring-payments -->
-
-					<!-- wp:spacer {"height":"20px"} -->
-					<div style="height:20px" aria-hidden="true" class="wp-block-spacer"></div>
-					<!-- /wp:spacer -->
-
-					<!-- wp:paragraph {"align":"left","style":{"typography":{"fontSize":18}}} -->
-					<p class="has-text-align-left" style="font-size:18px">${ translate(
-						`Describe your item. You can add a few lines here that describe the item you're selling. Or just delete this block if you don't need it!`
-					) }</p>
-					<!-- /wp:paragraph -->
-
-					<!-- wp:social-links {"iconColor":"foreground-dark","iconColorValue":"#101010","className":"is-style-logos-only"} -->
-					<ul class="wp-block-social-links has-icon-color is-style-logos-only">
-						<!-- wp:social-link {"service":"facebook"} /-->
-
-						<!-- wp:social-link {"service":"twitter"} /-->
-					</ul>
-					<!-- /wp:social-links -->
-				</div>
-				<!-- /wp:column -->
-			</div>
-			<!-- /wp:columns -->
+			<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"100px","bottom":"100px"}},"color":{"background":"#ecf2f3"}},"layout":{"inherit":true}} -->
+			<div class="wp-block-group alignfull has-background" style="background-color:#ecf2f3;padding-top:100px;padding-bottom:100px"><!-- wp:columns {"align":"wide"} -->
+			<div class="wp-block-columns alignwide"><!-- wp:column -->
+			<div class="wp-block-column"><!-- wp:image {"id":4390,"sizeSlug":"large","linkDestination":"none"} -->
+			<figure class="wp-block-image size-large"><img src="https://dotcompatterns.files.wordpress.com/2022/02/pexels-cottonbro-7026777.jpg?w=683" alt="" class="wp-image-4390"/></figure>
+			<!-- /wp:image --></div>
+			<!-- /wp:column -->
+
+			<!-- wp:column {"style":{"spacing":{"padding":{"right":"40px","left":"40px"}}}} -->
+			<div class="wp-block-column" style="padding-right:40px;padding-left:40px"><!-- wp:heading {"level":1,"style":{"typography":{"fontSize":"40px"},"spacing":{"margin":{"top":"0px"}}}} -->
+			<h1 id="item-name" style="font-size:40px;margin-top:0px">Item Name</h1>
```

Comment:
Shouldn't the 'Item Name' string be wrapped in a `translate` call?

Reply:
I don't *think* so but I'm not entirely clear on that. @kwight do you recall where we settled on the block translations issue?